R/update_pbp_participation.R

```diff
@@ -108,9 +116,22 @@ pbp_participation <-
         offense_players,
         defense_players,
         n_offense,
-        n_defense
+        n_defense,
+        air_yards,
+        time_to_throw,
+        was_pressure,
+        route,
+        dplyr::any_of(c('defense_man_zone_type','defense_coverage_type'))
       )
 
+    if(!('defense_man_zone_type' %in% colnames(plays))) {
```

Comment:
is a pattern like this a bit more extensible/clear? Also provides some type safety for other cols as well

https://github.com/nflverse/ngsscrapR/blob/master/R/roster.R#L181-L222

https://github.com/nflverse/ngsscrapR/blob/3c2473a7beb536426dcf2352d9e95d3b56133580/R/roster.R#L176